tayoto_mikaellaantonette/script.js

```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
 sortDescButton.addEventListener("click", function () {
   sortComments(false);
 });
+
+function searchCountry() {
+  var countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5